db/migrate/20231004120417_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: :users }
+      t.string :title
+      t.string :text
```

Comment:
- [ ]  [optional] @SanaTameem I would recommend changing the datatype from string to text, this is because text allows for a large input compared to string and would be suitable in this instance.